### DISCUSSION TITLE
Can I accumulate/get/read IOMMU IOTLB stats?

### DISCUSSION THREAD

**Discussion by pearls-of-agrippa** [opcm/pcm#267] [Q&A] [answered]
I am investigating an issue where the IOMMU may be getting pushed into a corner by the host page size that is in use.  Is there a way to monitor/examine IOMMU things like IOTLB using this tool suite?

**Comment by opcm** [comment, marked as answer]
pcm-iio monitors IO TLB hits and misses:

see pcm-iio event descriptions:

https://github.com/opcm/pcm/blob/0a178ec2573821e3c08beb6bd76808ebb912bc76/opCode.txt#L20

https://github.com/opcm/pcm/blob/0a178ec2573821e3c08beb6bd76808ebb912bc76/opCode.txt#L21
